### PR TITLE
Change README.md import from ms-rest-js to ms-rest

### PR DESCRIPTION
**Pull request by daschult**

### CODE REVIEW

**Comment by olydis** [comment]
# 🤖 AutoRest automatic publish job 🤖
~~~ Haskell
> will publish once PR gets merged
~~~

**Comment by olydis** [comment]
# 🤖 AutoRest automatic feature coverage report 🤖
*feature set version 2.4.2*

## ❌️ General: 0%

no tests were run for this category


## ✔️ Azure: 100%

**Comment by olydis** [comment]
# 🤖 AutoRest automatic feature coverage report 🤖
*feature set version 2.4.2*

## ❌️ General: 0%

no tests were run for this category


## ✔️ Azure: 100%

**Comment by olydis** [comment]
# 🤖 AutoRest automatic feature coverage report 🤖
*feature set version 2.4.2*

## ❌️ General: 0%

no tests were run for this category


## ✔️ Azure: 100%

File: src/autorest.nodejs.csproj [hunk 1]
```diff
@@ -79,5 +79,8 @@
   <ItemGroup>
     <DotnetRazor Remove="vanilla\Templates\ReadmeTemplate.cshtml" />
   </ItemGroup>
+  <ItemGroup>
```
Comment (RikkiGibson) [inline src/autorest.nodejs.csproj]: What does this do?

Comment (daschult) [reply src/autorest.nodejs.csproj]: Good catch. I originally thought I would need a PackageJson template specifically for Azure, but I don't think my final implementation needed it. I'll check and remove it if it isn't needed.

**Comment by olydis** [comment]
# 🤖 AutoRest automatic feature coverage report 🤖
*feature set version 2.4.2*

## ❌️ General: 0%

no tests were run for this category


## ✔️ Azure: 100%

**Comment by olydis** [comment]
# 🤖 AutoRest automatic feature coverage report 🤖
*feature set version 2.4.2*

## ❌️ General: 0%

no tests were run for this category


## ✔️ Azure: 100%

**Pull request merged**